addons/web/static/src/legacy/js/fields/basic_fields.js

```diff
@@ -1741,6 +1741,7 @@ var UrlWidget = InputField.extend({
             href: href,
             target: '_blank',
         });
+        this.el.textContent = '';
```

Comment:
Why do you need this ? It's "dangerous" to touch these files :)

Reply:
Hey @SimonGenin, (sorry for late reply)

So, in Recruitment app, when we create an tracker, let's say we first choose Facebook, then Twitter... The URL parameter of Twitter does not replace Facebook URL, instead it is added to it. All of this before we save it. So we see long unmeaningful URL. That's why this. What do you think? Can we do it some other way?

Reply:
Fix seems ok, but you should write a test